action-whitelist/approved.json

```diff
@@ -0,0 +1,55 @@
+[
+    {
+      "actionLink": "actions/checkout",
+      "actionVersion": "ec3a7ce113134d7a93b817d10a8272cb61118579"
+    },
+    {
+      "actionLink": "actions/cache",
+      "actionVersion": "c64c572235d810460d0d6876e9c705ad5002b353"
+    },
+    {
+      "actionLink": "actions/create-release",
+      "actionVersion": "v1"
```

Comment:
Prefer to use commit hash even for GitHub's actions, so that developers don't need to think "does this one use version tag or commit hash"?